src/components/BookItem.js

```diff
@@ -27,5 +38,16 @@ function BookItem() {
     </li>
   );
 }
+// item_id: 'item1',
+// title: 'The Great Gatsby',
+// author: 'John Smith',
+// category: 'Fiction',
```

Comment:
- [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code, this will make your codebase look neat and readable 😉